app/views/layout.html

```diff
@@ -68,6 +68,26 @@
     }
 %}
 
```

Comment:
I think the below section could be in the charts-home or charts-layout views, just to save milliseconds building it for the other pages. :)